hack/lib/tracing.bash

```diff
@@ -73,6 +73,10 @@ spec:
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
+      securityContext:
```

Comment:
Not sure if things will fail on 4.12 by default (need to test), It would fail if we set `restricted` on the tracing namespace for sure though. However, it does not do any harm I suppose.